### ISSUE TITLE
Improve article 2012.05155 - \eqref doesn't seem to work

### ISSUE THREAD

**Issue by pelegm** [dginev/ar5iv#33]
For example, after equation (1) here:
https://ar5iv.org/html/2012.05155#S1.E1

**Problem details**

An equation reference is expected. Instead, it reads `\reftagform@1`.

**Desktop**
 - OS: Ubuntu 20.04
 - Browser: Firefox 96.0

**Screenshot**
![Screenshot from 2022-02-02 12-58-05](https://user-images.githubusercontent.com/3422626/152210704-3e9a267a-6bfe-4306-bc82-3331d7e96985.png)

**Comment by pelegm** [comment]
Just updating that the issue still seem to exist.

**Comment by dginev** [comment]
@pelegm Indeed. Until support for `tabto.sty` lands in either [latexml](https://github.com/brucemiller/LaTeXML/) or the extended [ar5iv-bindings](https://github.com/dginev/ar5iv-bindings/), this issue will persist.

Looking at our build system, `tabto.sty` is used in 0.02% of arXiv articles, or 463 total. This means we won't get to this soon.

**Comment by pelegm** [comment]
Thanks so much for this update!
